cumulusci/tasks/salesforce/tests/test_CreateCommunity.py

```diff
@@ -0,0 +1,106 @@
+import json
+import mock
+import responses
+import unittest
+from datetime import datetime
+from cumulusci.tasks.salesforce import CreateCommunity
+from cumulusci.core.exceptions import SalesforceException
+from .util import create_task
+
+
+task_options = {
+    "name": "Test Community",
+    "description": "Community Details",
+    "template": "VF Template",
+    "url_path_prefix": "test",
+}
+
+
+class test_CreateCommunity(unittest.TestCase):
+    @responses.activate
+    def test_creates_community(self):
+        cc_task = create_task(CreateCommunity, task_options)
+        cc_task.sf = mock.Mock()
+        cc_task.sf.restful.return_value = {
+            "communities": [{"name": "Test Community", "id": "000000000000000"}]
+        }
```

Comment:
I generally prefer to use `responses` to provide mock responses, rather than mocking the library call which makes the request. That way if the library ever changes what request it's producing or how it processes the response, our tests will let us know.

Open to pushback if that's hard in this particular case for some reason.